Makefile

```diff
@@ -103,6 +103,11 @@ clean-migrations-compose:
 
 clean: clean-protos clean-migrations-compose
 
+smithyctl/bin:
```

Comment:
I think we have this already defined as `smithyctl/cmd/bin` - you might want to remove that

Reply:
that one you mention is for the old smithyctl binary